arxiv/browse/util/search_authors.py

```diff
@@ -0,0 +1,166 @@
+"""Produces search service query strings for authors."""
```

Comment:
If this is about url generation (looks like it, in essence), it should be in ``arxiv.urlize`` with the arXiv ID and DOI link generation code.